webapp/channels/src/components/form_error.tsx

```diff
@@ -11,71 +11,70 @@ type Props = {
     iconClassName?: string;
     margin?: boolean;
     errors?: React.ReactNode[];
-}
+};
 
-export default class FormError extends React.PureComponent<Props> {
-    public static defaultProps = {
-        error: null,
-        errors: [],
-    };
-    public render() {
-        const {error = null, errors = [], iconClassName, margin, textClassName, type} = this.props;
-
-        if (!error && errors.length === 0) {
-            return null;
-        }
+const FormError: React.FC<Props> = ({
+    error = null,
+    errors = [],
+    iconClassName,
+    margin,
+    textClassName,
+    type,
+}) => {
+    if (!error && errors.length === 0) {
+        return null;
+    }
 
-        // look for the first truthy error to display
-        let message = error;
+    // look for the first truthy error to display
+    let message = error;
 
-        if (!message) {
-            for (const err of errors) {
-                if (err) {
-                    message = err;
-                }
+    if (!message) {
+        for (const err of errors) {
+            if (err) {
+                message = err;
+                break;
```

Comment:
Probably not a big issue, but this changes slightly the original behavior. It used to show the last error, now it shows the first. Since `errors` shouldn't be long in any case, unless you have a strong opinion about it, let's remove the break.